docs/pages/versions/unversioned/guides/configuring-statusbar.md

```diff
@@ -13,35 +13,57 @@ The configuration for Android status bar lives under the `androidStatusBar` key
 This option can be used to specify whether the status bar content (icons and text in the status bar) is light, or dark. Usually a status bar with a light background has dark content, and a status bar with a dark background has light content.
 
 The valid values are:
-
 - `light-content` - The status bar content is light colored (usually white). This is the default value.
 - `dark-content` - The status bar content is dark colored (usually dark grey). This is only available on Android 6.0 onwards. It will fallback to `light-content` in older versions.
 
+> Note: Be aware that choosing `light-content` and having `SplashScreen` image that is very light (close to `white` color) and/or having `backgroundColor` of your app set to `white` (or close to `white` color) may result in status bar's icons blending in and wouldn't be visible.
```

Comment:
```suggestion
> Note: If you choose `light-content` and have either your app's `SplashScreen` or `backgroundColor` also set to a light color, the status bar icons may blend in and not be visible.
```

Reply:
thanks for adding this note! we should follow up in a future PR to finish off https://github.com/expo/expo/pull/5230 so the status bar can be customized for the splash screen

Reply:
@cruzach, I've described what's on my mind a little more, because I'm not sure that your suggestion is clearly understandable:
```
Note: If you choose `light-content` and have either your app's `SplashScreen` or `backgroundColor` also set to a light color, the status bar icons may blend in and not be visible.
```
What does it mean to `have your app's SplashScreen set to a light color` 🤔 😛 
Please check out my new description for this 🙏

Reply:
Your new description is perfect 👍 

Sorry, I should've been clearer. Instead of "have your app's SplashScreen set to a light color" I should've said "having a light-colored SplashScreen image"